flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/channel/ChannelStateWriteRequestExecutorImplTest.java

```diff
@@ -119,15 +135,19 @@ public void testCleanup() throws IOException {
     public void testIgnoresInterruptsWhileRunning() throws Exception {
         TestRequestDispatcher requestProcessor = new TestRequestDispatcher();
         LinkedBlockingDeque<ChannelStateWriteRequest> deque = new LinkedBlockingDeque<>();
-        try (ChannelStateWriteRequestExecutorImpl worker =
-                new ChannelStateWriteRequestExecutorImpl(TASK_NAME, requestProcessor, deque)) {
+        ChannelStateWriteRequestExecutorImpl worker =
+                new ChannelStateWriteRequestExecutorImpl(requestProcessor, deque, JOB_ID, 5);
```

Comment:
Hi @pnowojski , I'm not sure should these `channel state` classes be moved to the `flink-streaming-java` module?

All checkpoint-related configurations are defined in the `flink-streaming-java` module, and these `channel state` classes are defined in the `flink-runtime` module. 

These classes didn't read any configuration before, so they are well. However, It's hard to read some configurations during unit test due to `flink-runtime` doesn't depend on the `flink-streaming-java` module.